logexporter/exporter.go

```diff
@@ -20,20 +18,34 @@ type Exporter interface {
 	Wait()
 }
 
-func NewExporter(logger *logrus.Logger, client castai.Client) Exporter {
-	return &exporter{
-		logger: logger,
-		client: client,
-		wg:     sync.WaitGroup{},
-	}
```

Comment:
Maybe LogSender than?

Reply:
I think linter was complaining that there is a tautology. It was saying that `logexporter.Sender` is better than `logexporter.LogSender`.
I will double check and change if it doesn't complain.